sdks/python/apache_beam/transforms/core.py

```diff
@@ -324,11 +324,12 @@ def __init__(self, fn):
       raise TypeError('Expected a callable object instead of: %r' % fn)
 
     self._fn = fn
-    if _fn_takes_side_inputs(fn):
```

Comment:
No the DoFnRunner inspects the function so CallableWrapper can just let it inspect the `fn` directly.